src/helpers/getCourses.ts

```diff
@@ -0,0 +1,13 @@
+import { mockedCoursesList as data } from 'src/constants';
```

Comment:
Feel free to keep it, but it's unnecessary

 ```suggestion
import { mockedCoursesList } from 'src/constants';
```